src/main/java/org/opentestsystem/ap/ims/service/ItemBankService.java

```diff
@@ -86,10 +85,23 @@ public void updateItem(String itemId, Item item) {
      */
     public void commitItemChanges(String itemId) {
         if (StringUtils.isBlank(itemId)) {
-            throw new ValidationException("Item ID is required when merging an item");
+            throw new ValidationException("Item ID is required");
         }
         final ItemBankUser user = securityUtil.getItemBankUser();
         itemBankRepository.commitItemChanges(user, itemId);
 
     }
+
+    /**
+     * Delete's an item from the item bank.
+     *
+     * @param itemId The item to delete.
+     */
+    public void deleteItem(String itemId) {
+        if (StringUtils.isBlank(itemId)) {
+            throw new ValidationException("Item ID is required");
+        }
+        final ItemBankUser user = securityUtil.getItemBankUser();
+        itemBankRepository.deleteItem(user, itemId);
```

Comment:
Yes this would be a good place to throw.  I want to keep custom Exception classes to a minimum but this seems like a good place for RecordNotFoundException. Throw it here and the ExceptionAdvice will handle RecordNotFound and return the appropriate status code 404 and the same json all other exceptions thrown i.e. the ErrorMessage class.